src/coreclr/vm/precode.cpp

```diff
@@ -133,6 +133,9 @@ MethodDesc* Precode::GetMethodDesc(BOOL fSpeculative /*= FALSE*/)
     TADDR pMD = (TADDR)NULL;
 
     PrecodeType precodeType = GetType();
+#ifdef __wasm__
+    pMD = *(TADDR*)(m_data + OFFSETOF_PRECODE_TYPE + 4);
```

Comment:
Why? Can you add a comment, either an explanation or a special FIXME?

Reply:
I added another offset for the MD to make it clear and also added comment to the offsets defines